### ISSUE TITLE
Need sounds with slower modulation for "Slow" speed setting

### ISSUE THREAD

**Issue by jbphet** [phetsims/molecules-and-light#289]
In https://github.com/phetsims/molecules-and-light/issues/262, the ability to set the sim into a "slow motion" mode was added.  The sounds that are used to indicate vibration and rotation (clockwise and counterclockwise) are timed to match the rate at which the normal (i.e. non-slow-motion) behavior operates.  We discussed this in the Feb 25 2020 sound design meeting and decided that we'd like to keep the basic pitch of these sounds the same in slow motion, but change the rate of the low-frequency modulation to match the slow motion behavior.  Since Web Audio doesn't include the ability the set pitch-independent playback rate, this will require new sound clip files.

The code that sets the slow motion rate looks like this (from `PhotonAbsorptionModel`:
```js
      if ( this.slowMotionProperty.get() ) {
        dt = dt / 3;
      }
```

The means the modulation rate will need to be 1/3 of the normal rate.

@Ashton-Morris - assigning to you.  In conjunction with https://github.com/phetsims/molecules-and-light/issues/284, which is about investigating stereo and panning, we will need the following new files:

- [ ] vibration with 1/3 modulation rate
- [ ] stereo clockwise rotation, normal modulation rate
- [ ] stereo counterclockwise rotation, normal modulation rate
- [ ] stereo clockwise rotation, 1/3 modulation rate
- [ ] stereo counterclockwise rotation, 1/3 modulation rate

**Comment by Ashton-Morris** [comment]
I have created files that are 1/3 the current rate for the "Slow Motion" mode. 

[They can be found here.](https://drive.google.com/open?id=1UdeAk6MxXEZMSj2Nt6tDi35R-jej5ppM)

**Comment by emily-phet** [comment]
@jbphet Please prioritize this issue in preparation for Tuesday's design meeting. Our goal is to get MAL design complete by the end of the month, so we'll need to have an initial listen and consider next steps soon for "slow motion" sound in order to meet that goal. 

Note - spatialization is secondary.

**Comment by jbphet** [comment]
The sounds have been integrated, and a dev version is available at https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.37/phet/molecules-and-light_en_phet.html.

**Comment by jbphet** [comment]
We reviewed the dev version linked in the comment just above in the 3/10/2020 sound design meeting.  The general consensus was that the changes are good and we should go with them.  Here are some other notes from the discussion:

+ @Ashton-Morris noted that the slower photon sounds hadn't been incorporated.  I hadn't realized that they were supposed to be, but after some discussion, we decided that slower sounds weren't desirable in this case since the emission is essentially instantaneous and we are not (yet) using spatialized sounds for these.
+ The break-apart and glow sounds don't change in slow-motion mode, and the design team is fine with this.
+ It was suggested that we consider taking some of the higher harmonics off of the glow sound.  I'll create a separate issue for that.
+ We will need stereo versions of these slow motion sounds to try out for https://github.com/phetsims/molecules-and-light/issues/284.

The work necessary for the notes listed above is all covered under other GitHub issue, so I'm going to close this one.
